.travis.yml

```diff
@@ -27,3 +27,29 @@ matrix:
       script:
         - make install
         - danger-kotlin ci
+    - os: osx
+      osx_image: xcode10
+      install:
+        - curl -s "https://get.sdkman.io" | bash
+        - source ~/.bash_profile
+        - sdk install kscript
+        - sdk install gradle 5.2.1
```

Comment:
If you remove the version it **should** take the latest one, that is the one we most care about probably if we take the brew way

Reply:
i always use to use fixed versions, if there is an updated version could break something and we not have any control of that. I think is a good idea use known versions. BTW, with gradle 5 there is some problems on kotlin native.. i'm working on that

Reply:
You are right, but in this specific case, given we are going to use brew as explained in the article @orta posted here https://github.com/danger/kotlin/issues/13#issuecomment-460879054, brew doesn't use versions, this means that if we add gradle as dependency who installs danger-kotlin gets it with the latest version available, then I would suggest to test against the one the people will get